tools/topology/topology2/cavs-sdw.conf

```diff
@@ -42,6 +42,10 @@
 <output_audio_format.conf>
 <dmic-default.conf>
 <hdmi-default.conf>
+<platform/intel/ssp_aux_config.conf>
+<ssp.conf>
+<bt-default.conf>
+<bt-generic.conf>
```

Comment:
@udaymb can't do this. This will add BT to every sdw topology

Reply:
Updated the conf file accordingly